logstash/class-logger.php

```diff
@@ -565,7 +565,7 @@ public static function maybe_wp_debug_log_entries( array $entry ): void {
 	 * @param array $entry Data.
 	 */
 	public static function wp_debug_log( array $entry ): void {
-		if ( defined( 'VIP_GO_ENV' ) && VIP_GO_ENV ) {
+		if ( defined( 'VIP_GO_ENV' ) && constant( 'VIP_GO_ENV' ) !== 'local' ) {
```

Comment:
I think we can remove this check since we already do the checking above.

Reply:
Good point. Removed here: https://github.com/Automattic/vip-go-mu-plugins/pull/5727/commits/aa4ca76060ff113777195cf46df5f9f7c3db19cf

Note that now we'll log if people call `wp_debug_log` manually, and before it would skip it if it wasn't in the local env.

Reply:
I'm not seeing usages of it outside of vip-go-mu-plugins, so we should be all set here.